lib/context.ml

```diff
@@ -65,3 +52,15 @@ let save () =
 
 let restore () =
   match !context with Some ctx -> Cairo.restore ctx.ctx | None -> fail ()
+
+let init_context background_color line_width ((x, y) : int * int) axes =
```

Comment:
We shouldn't only have a type signature for one parameter.
That should be in the `.mli` file.

Also, if these are the dimensions of the canvas, `w` and `h` would be more consistent names - `x`, `y` for point coordinates, and `w`, `h` for canvas size

Reply:
Ah yeah this was left over from the type inference system insisting that these were floats, I should just be able to remove it.

Will rename x,y as well